wrapper/src/main/java/software/amazon/jdbc/dialect/MySQLDialect.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package software.amazon.jdbc.dialect;
+
+import software.amazon.jdbc.exceptions.ExceptionHandler;
+import software.amazon.jdbc.exceptions.MySQLExceptionHandler;
+
+public class MySQLDialect implements DatabaseDialect {
```

Comment:
This dialects makes sense for Aurora MySQL clusters. RDS MySQL and vanilla MySQL databases don't support queries mentioned here. Does it mean that there should be two separate dialects, one for Aurora, another one for MySQL?

Reply:
It did expose the MariaDB gap.